include/render/fx_renderer/fx_renderer.h

```diff
@@ -0,0 +1,83 @@
+#ifndef _FX_OPENGL_H
+#define _FX_OPENGL_H
+
+#include <GLES2/gl2.h>
+#include <GLES2/gl2ext.h>
+#include <stdbool.h>
+#include <wlr/render/egl.h>
+#include <wlr/render/wlr_texture.h>
+#include <wlr/util/addon.h>
+
+#include "wlr/util/box.h"
+
+enum fx_tex_shader_source {
+	SHADER_SOURCE_TEXTURE_RGBA = 1,
+	SHADER_SOURCE_TEXTURE_RGBX = 2,
+	SHADER_SOURCE_TEXTURE_EXTERNAL = 3,
+};
+
+struct decoration_data {
+	float alpha;
+	int corner_radius;
+};
```

Comment:
We should remove this before merging (need to rethink how we handle decoration data)